py3status/modules/sway_idle.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Display sway inhibit idle status.
+
+This Module shows an indicator, if an idle is inhibited by an inhibitor.
+For more information about inhibit idle see `man 5 sway`
+
+Configuration parameters:
+    cache_timeout: How often we refresh this module in seconds (default 1)
+    format: Display format (default 'Inhibit Idle: [\?if=inhibit_idle=True True]|False')
+
+Format placeholders:
+    {inhibit_idle} Returns 'True' if idle is inhibited, 'False' else.
+
+Example:
+
+```
+sway_idle {
+    format = "[\?if=inhibit_idle=True Idle Inhibited]"
+}
+```
+
+@author Valentin Weber <valentin+py3status@wv2.ch>
+@license BSD
+
+SAMPLE OUTPUT
+{full_text': 'Inhibit Idle: True'}
+"""
+
+
+class Py3status:
+
+    # available configuration parameters
+    cache_timeout = 1
+    format = "Inhibit Idle: [\?if=inhibit_idle=True True]|False"
```

Comment:
(Suggestion) "Inhibit Idle: {inhibit_idle}"

Reply:
@weberval ping, did you consider this please?

Reply:
Sorry for not replying, I was very busy this week.

The complicated format was meant as an explanation about how to use the `inhibit_idle` bool, but I can change it.